cmd/deploy/deploy.go

```diff
@@ -131,7 +132,12 @@ func Deploy(ctx context.Context) *cobra.Command {
 			// This is needed because the deploy command needs the original kubeconfig configuration even in the execution within another
 			// deploy command. If not, we could be proxying a proxy and we would be applying the incorrect deployed-by label
 			os.Setenv(model.OktetoSkipConfigCredentialsUpdate, "false")
-
+			if options.ManifestPath != "" {
```

Comment:
Does this take into consideration the case in which the manifest is within a `.okteto` folder? I think that is checked automatically and in that case `ManifestPath` would be empty

Reply:
Yes it will, but only if the user specify the main file (not the one into the `.okteto` folder). If the user run `okteto deploy -f .okteto/okteto.yaml` it will run inside the `.okteto` folder.

Reply:
yeah, but I mean, If I have stored the manifest in `.okteto/okteto.yml`, I would probably have defined the commands and file paths taking that into account. When I run `okteto deploy` (without the - f parameter), it will detect correctly the manifest because we check that path but the commands will probably fail because it is being executed from the current directory instead of doing it from `.okteto`

Reply:
After talking with @pchico83, we have agreed that the `.okteto` directory is a special case which makes your manifests be in the root directory but hidden. Before we merge we have to make that if the user specifies the path `in .okteto` we do not deploy from that folder, but from the previous one from that